packages/quantic/force-app/main/default/lwc/descriptionStrengthIndicator/descriptionStrengthIndicator.html

```diff
@@ -0,0 +1,21 @@
+<template>
+  <div class="slds-grid">
+    <div class="slds-p-top_x-small">
+      <lightning-progress-ring size="large" value={progress} variant={variant}></lightning-progress-ring>
+    </div>
+    <div class="slds-col">
+      <h3 class="slds-text-title_bold slds-p-left_small">{message}</h3>
+      <lightning-accordion allow-multiple-sections-open class="accordion">
+        <lightning-accordion-section label={helpLabel}>
+          <div class="slds-m-left_large">
+            <slot name="content">
+              Since when is this problem affecting you?<br />
```

Comment:
If you're planning on having custom labels in the cookbook as well (which might be a good idea indeed) then these strings should be labels as well.